lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/CommentGradingPaneProducer.java

```diff
@@ -204,14 +204,15 @@ public void fillComponents(UIContainer tofill, ViewParameters viewparams, Compon
 		}
 		
 		if (notSubmitted.size() > 0) {
-		    List<String> missing = new ArrayList<String>();
-		    for (String userId: notSubmitted) {
-			try {
-			    missing.add(UserDirectoryService.getUser(userId).getDisplayName());
-			} catch (Exception e) {
-			    missing.add(userId);
+			List<String> missing = new ArrayList<String>();
```

Comment:
This spacing seems off using 8 vs 4 maybe

Reply:
The spacing of that file is a mess, I think is fine, if you look before:
![image](https://user-images.githubusercontent.com/8653754/75752059-38b9d800-5d28-11ea-92b3-00e833630dd9.png)

And after:
![image](https://user-images.githubusercontent.com/8653754/75752079-42434000-5d28-11ea-941a-bacc33bd601d.png)

After the changes the spacing is consistent using tabs.